README.yaml

```diff
@@ -5,40 +5,99 @@
 #
 
 # Name of this project
-name : Terraform Module Template
+name : Terraform Module for Google Cloud Pub/Sub
 
 # License of this project
 license: "APACHE"
 
 # Canonical GitHub repo
-github_repo: clouddrove/terraform-module-template
+github_repo: clouddrove/terraform-gcp-pubsub
 
 # Badges to display
 badges:
   - name: "Latest Release"
-    image: "https://img.shields.io/github/release/clouddrove/terraform-module-template.svg"
-    url: "https://github.com/clouddrove/terraform-module-template/releases/latest"
+    image: "https://img.shields.io/github/release/clouddrove/terraform-gcp-pubsub.svg"
+    url: "https://github.com/clouddrove/terraform-gcp-pubsub/releases/latest"
   - name: "tfsec"
-    image: "https://github.com/clouddrove/terraform-module-template/actions/workflows/tfsec.yml/badge.svg"
-    url: ""
-  - name: "Licence"
+    image: "https://github.com/clouddrove/terraform-gcp-pubsub/actions/workflows/tfsec.yml/badge.svg"
+    url: "https://github.com/clouddrove/terraform-gcp-pubsub/actions/workflows/tfsec.yml"
+   - name: "Licence"
     image: "https://img.shields.io/badge/License-APACHE-blue.svg"
     url: "LICENSE.md"
+  - name: "Changelog"
+    image: "https://img.shields.io/badge/Changelog-blue"
+    url: "CHANGELOG.md"
 
 prerequesties:
```

Comment:
Only prerequesties where is `provider` field
plz refere following readme.yaml https://github.com/clouddrove/terraform-aws-eks/blob/master/README.yaml

Reply:
done sir